frontend/src/containers/ReportStep/index.tsx

```diff
@@ -111,6 +121,23 @@ const ReportStep = ({ handleSave }: ReportStepProps) => {
   const onlySelectClassification = useAppSelector(isOnlySelectClassification);
   const isSummaryPage = useMemo(() => pageType === REPORT_PAGE_TYPE.SUMMARY, [pageType]);
 
+  const mapDateResult = (descendingDateRanges: DateRange, reportInfos: IReportInfo[]) =>
+    descendingDateRanges.map(({ startDate, endDate }) => {
+      const reportData = reportInfos.find((singleResult) => singleResult.id === startDate)?.reportData ?? null;
+      return {
+        startDate: startDate,
+        endDate: endDate,
+        overallMetricsCompleted: reportData?.overallMetricsCompleted ?? false,
+        boardMetricsCompleted: reportData?.boardMetricsCompleted ?? null,
+        doraMetricsCompleted: reportData?.doraMetricsCompleted ?? null,
```

Comment:
`overallMetricsCompleted` potentially infers to a `Boolean`, however `boardMetricsCompleted` & `doraMetricsCompleted` infer to `Nullable<boolean>`. How about using a same type?